src/devices/bus/isa/myb3k_fdc.h

```diff
@@ -14,9 +14,23 @@
 #include "machine/wd_fdc.h"
 #include "formats/imd_dsk.h"
 
+class isa8_myb3k_fdc471x_device
```

Comment:
Please don't use `_device` suffix for things that are not `device_t` implementations.  It's horribly confusing.

Reply:
ok, I'll fix

Reply:
Fixed